.circleci/config.yml

```diff
@@ -46,4 +23,5 @@ jobs:
             - run: yarn lint
             - run: yarn prettier:ci
             - setup_remote_docker # Expose the docker daemon so that it can be used in tests.
-            - run: yarn test
+            - run: yarn test:rfqt
+            - run: yarn test:rest
```

Comment:
why split these?

Reply:
These must be split because we can't cleanly tear down `app` right now. This will cause it to try to reconnect to Mesh and cause a whole host of other issues. The easiest way to fix it was to separate the tests so that there is only a single suite with an `app` instance. For all future suites, we should use the new flows, but from speaking to the maker team, this may be somewhat difficult for the RFQT tests in the near term.